go test currently fails

```
~ »  go get -v github.com/TileDB-Inc/TileDB-Go
...

~ » go test -v github.com/TileDB-Inc/TileDB-Go
# github.com/TileDB-Inc/TileDB-Go
TileDB-Go/config_test.go:5:2: cannot find package "github.com/stretchr/testify/assert" in any of:
	/usr/local/Cellar/go/1.10.3/libexec/src/github.com/stretchr/testify/assert (from $GOROOT)
	/Users/jacobbolewski/go/src/github.com/stretchr/testify/assert (from $GOPATH)
FAIL	github.com/TileDB-Inc/TileDB-Go [setup failed]
```

--- a/README.md
+++ b/README.md
@@ -22,6 +22,18 @@ To install these bindings you can use go get:
  go get -v github.com/TileDB-Inc/TileDB-Go
 ```
 
+To install package test dependencies:
+
+```bash
+go get -vt github.com/TileDB-Inc/TileDB-Go
+```
+
+Package tests can be run with:
+
+```bash
+go test github.com/TileDB-Inc/TileDB-Go
+```
+
 ## Example Usage
 
 Below is a small example using vfs functionality. Additionale examples are

